src/engine/platform/opl.cpp

```diff
@@ -1829,14 +1840,36 @@ void DivPlatformOPL::setFlags(const DivConfig& flags) {
         case 0x04:
           chipClock=15000000.0;
           break;
+        case 0x05:
+          chipClock=33868800.0;
```

Comment:
Where does this clock come from?

Reply:
datasheet said it's master clock frequency (input) is 33.8688MHz.

Reply:
44100×768